src/transformStateWithClones.js

```diff
@@ -66,6 +66,30 @@
  */
 function transformStateWithClones(state, transforms) {
   // write code here
+  const result = [];
+  const clone = { ...state };
+
+  for (const transform of transforms) {
```

Comment:
```suggestion
  for (const { operation, properties } of transforms) {
```
Можно использовать operation, properties вместо transform.properties,  transform.properties.